fix(remix-react): fix `fetcher.formEncType` (#6810)
remix-run/remix: .changeset/chatty-pears-float.md

---
"@remix-run/react": patch
---

> Changed the type of formEncType on useFetcher and useFetchers to be FormEncType from react-router-dom instead of strings

Reviewer: ```suggestion
Narrowed the type of `fetcher.formEncType` to use `FormEncType` from `react-router-dom` instead of `string`
```
